### ISSUE TITLE
Tantros-revealing mods bug

### ISSUE THREAD

**Issue by itush20** [Anuken/Mindustry#9459]
### Platforms

Android

### Build

offical release build 146

### Issue

I know this is for a mods bug category and not vannila category but still...
Well I heard once about a secret planet named tantros that you get from cheat code in pc.
'Cause Im mobile player, I wanted to Install one of the three mods based around tantros (picture 1)
![Screenshot_20240116-213641_Mindustry](https://github.com/Anuken/Mindustry/assets/126261466/940fdf51-990d-4d85-b5c5-0d238755e6a5)
 so **I installed 3 (tanteria)** then got tired from it.
**I deleted it and installed 1 (tantros-test).**
It was **t**antros with small letter + 8 blocks, lead, dark sand and a new resource.
(no drills 💀) Was bad.
**I deleted it too** then could see the planet name on list,  **but all the sectors and planet were sussly transparent.** also the sector renamed to "10" that wasnt a name in any mod I installed.
then Ive tried to **Install back 3 (tanteria)** to see what in it again (the transparency was ugly so...) ...
**Then I restarted and opened capmaign.
The moment I navigated to tantros the game crashed. Then, everytime I open the campaign it crashes.** 
**I deleted 3 once again (tanteria) and it fixed all. ** but... 
this transpa sussy thinge is still there...
![Screenshot_20240116-220233_Mindustry](https://github.com/Anuken/Mindustry/assets/126261466/a6293541-481f-49d7-ab4d-199a19daefb9)

**TLDR: the bug is only with 2 mods compiled on the same planet which wont happen in planets outa tantros**

nb: didnt touch 2 (tantros) cause it said early-access.

### Steps to reproduce

1. install tanteria
2. delete tanteria
3. install tantros-test
4. delete tantros test
5. tantros is still there with transp
6. i open a level (10)
7. dagger gurdian 🤨 on wave 6 🤨  (i deleted both mods and its still "playable")
8. install tanteria again
9. boom crash
10. delete tanteria again 
11. transp still there

### Mods used

https://github.com/twcash/tantros-test
tantros-test

https://github.com/jasmarkelina/tantros-mod
tanteria

https://github.com/yoru-kitsune/tantros
tantros
looks like a cool mod, didnt tested it yet
maybe it can work on here too

### Save file

forgot to say that i couldnt play at tanteria cause it have no starting sector (the reason i deleted it) thats actually a mod bug and not vanilla bug.

submerged on tanteria i think (locked) tried to enter then deleted

1 of the four maps in tantros-test (see Mods used)
i think algal feilds

10 on transparent tantros (no mods)

first time ever on bug report so idk how to export

### (Crash) logs

they dont support bin sorry

### Submission

- [X] I have updated to the latest release (https://github.com/Anuken/Mindustry/releases) to make sure my issue has not been fixed.
- [X] I have searched the closed and open issues to make sure that this problem has not already been reported.

**Comment by Anuken** [comment]
1. This is not a vanilla bug, so I cannot/will not fix it.
2. Even if I were able to fix it, crash bug reports **require logs**.
